main/main.go

```diff
@@ -469,6 +455,44 @@ func createApplicationRelayersForSourceChain(
 	return applicationRelayers, minHeight, nil
 }
 
+func createDeciderClient(host string, port *uint16) (*grpc.ClientConn, error) {
```

Comment:
Can you add a comment explaining the requirements for `host` and `port`? Namely what combinations result in a `nil` client, and which result in an error.

Reply:
addressed in https://github.com/ava-labs/awm-relayer/pull/344/commits/6a9c27686d03fe38b1e03933335186ab23cc91f6